DataScience/Readme.md

```diff
@@ -0,0 +1,40 @@
+# Mews data analyst task
+
+## Your task
+
+You have been given a task to look into data of one really nice hotel and find some answers and insights for following questions:
+
+1) What are the popular choices of booking rates (table rate, columns ShortRateName or RateName) for different segments of customers (table reservation, columns AgeGroup, Gender, NationalityCode) 
+
+2) What are the typical guests who do online check-in? Is it somehow different when you compare reservations created across different weekdays? (table reservation, IsOnlineCheckin column)
+
+3) Look at the average night cost per single occupied capacity. What guest segment is the most profitable per occupied space unit? And what guest segment is the least profitable?
+
+First, answer the questions using SQL and with your answer provide also the final SQL queries and output.
+
+Then, build a report/dashboard idealy using Power BI that also help answering the questions. You may also use python and build a jupyter notebook.
+
+Include the comments to present your solution and send it to vojta@mews.com. 
```

Comment:
could they somehow submit it via github? so you can have others looking at the tasks as well?

Reply:
We had a discussion with other members of data team - in this case the submitted solutions should be private. One of the key tasks is to get insights from the data and reading other's solutions would make it too easy for a candidate.
I'm not sure if github allows private pull requests on a public repository.